src/SingleBookingPage/BookingDetail.js

```diff
@@ -159,7 +159,9 @@ class BookingDetail extends React.Component<Props> {
         dataCy="nearestBooking"
         styles="width: 100%; padding:40px; background-color: #f5f7f9;"
       >
-        <Header booking={booking} isFuture={arrivalInfo.isFuture} />
+        <div className={arrivalInfo.isFuture ? 'upcoming' : 'past'}>
```

Comment:
I don't know if creating classes that are not used in CSS, but using it for testing is the best practice/approach.

Reply:
fixed